.github/workflows/main.yml

```diff
@@ -2,7 +2,7 @@ name: Python package
 on:
   push:
     tags:
-      - '[0-9]+\.[0-9]+\.[0-9]+'
+      - 'v[0-9]+\.[0-9]+\.[0-9]+'
```

Comment:
Just need to update the directions in `README.md` to reflect the change and this is good to go.